django_celery_beat/locale/ru/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,274 @@
+
+#: admin.py:169
+#, python-brace-format
+msgid "{0} task{1} {2} successfully disabled"
+msgstr "{0} задание{1} {2} успешно отключено"
+
+#: admin.py:175
+msgid "Disable selected tasks"
+msgstr "Отключить выбранные задания"
+
+#: admin.py:188
+#, python-brace-format
+msgid "{0} task{1} {2} successfully run"
+msgstr "{0} задание{1} {2} успешно выполнено"
+
+#: admin.py:194
+msgid "Run selected tasks"
+msgstr "Запустить выбранные задания"
+
+#: apps.py:15
+msgid "Periodic Tasks"
+msgstr "периодические задания"
+
+#: models.py:24
+msgid "Days"
+msgstr "Дни"
+
+#: models.py:25
+msgid "Hours"
+msgstr "Часы"
+
+#: models.py:26
+msgid "Minutes"
+msgstr "Минуты"
+
+#: models.py:27
+msgid "Seconds"
+msgstr "Секунды"
+
+#: models.py:28
+msgid "Microseconds"
+msgstr "Микросекунды"
+
+#: models.py:44
+msgid "event"
+msgstr "событие"
+
+#: models.py:47
+msgid "latitude"
+msgstr "широта"
+
+#: models.py:50
+msgid "longitude"
+msgstr "долгота"
+
+#: models.py:56
+msgid "solar event"
+msgstr "солнечное событие"
+
+#: models.py:57
+msgid "solar events"
+msgstr "солнечные события"
+
+#: models.py:101
+msgid "every"
+msgstr "каждый(ые)"
+
+#: models.py:103
+msgid "period"
+msgstr "период"
+
+#: models.py:109 models.py:232
+msgid "interval"
+msgstr "интервал"
+
+#: models.py:110
+msgid "intervals"
+msgstr "интервалы"
+
+#: models.py:130
+#, python-brace-format
+msgid "every {0.period_singular}"
+msgstr "каждый(ые) {0.period_singular}"
+
+#: models.py:131
+#, python-brace-format
+msgid "every {0.every} {0.period}"
+msgstr "каждый(ые) {0.every} {0.period}"
+
+#: models.py:142
+msgid "minute"
+msgstr "минута"
+
+#: models.py:143
+msgid "hour"
+msgstr "час"
+
+#: models.py:145
+msgid "day of week"
+msgstr "день недели"
+
+#: models.py:148
+msgid "day of month"
+msgstr "день месяца"
+
+#: models.py:151
+msgid "month of year"
+msgstr "месяц в году"
+
+#: models.py:157 models.py:236
+msgid "crontab"
+msgstr "расписание"
+
+#: models.py:158
+msgid "crontabs"
+msgstr "расписания"
+
+#: models.py:226
+msgid "name"
+msgstr "название"
+
+#: models.py:227
+msgid "Useful description"
+msgstr "Понятное описание"
+
+#: models.py:229
+msgid "task name"
+msgstr "название задания"
+
+#: models.py:236
+msgid "Use one of interval/crontab"
+msgstr "Используйте интервал или расписание"
+
+#: models.py:240
+msgid "solar"
+msgstr "солнечное событие"
+
+#: models.py:240
+msgid "Use a solar schedule"
+msgstr "Задайте солнечное событие"
+
+#: models.py:244
+msgid "JSON encoded positional arguments"
+msgstr "Позиционные аргументы, закодированные в JSON"
+
+#: models.py:247
+msgid "Keyword arguments"
+msgstr "Именованные аргументы"
+
+#: models.py:248
+msgid "JSON encoded keyword arguments"
+msgstr "Именованные аргументы, закодированные в JSON"
+
+#: models.py:251
+msgid "queue"
+msgstr "очередь"
+
+#: models.py:252
+msgid "Queue defined in CELERY_TASK_QUEUES"
+msgstr "Очередь, определенная в CELERY_TASK_QUEUES"
+
+#: models.py:255
+msgid "exchange"
+msgstr "обмен"
```

Comment:
This doesn't make any sense IMO. I wouldn't translate this, this is a term <http://docs.celeryproject.org/en/latest/userguide/routing.html#exchanges-queues-and-routing-keys>

The same for the routing key, TBH.